parking_permits_app/models.py

```diff
@@ -0,0 +1,386 @@
+import uuid
```

Comment:
Since `models.py` usually evolves quickly and manually drawn entity-relationship diagrams like the one you have provided go out of date quickly, do you think it makes sense to create a ticket for automatically generating a database diagram using some kind of a library?

Reply:
Let's get back to this at a later stage in the project when most critical components are stable and we can focus on polishing the details here and there. Right now, I think the diagram works quite nicely 👍🏼

Reply:
Ticker created for this: https://helsinkisolutionoffice.atlassian.net/browse/PV-166